frameit/lib/frameit/editor.rb

```diff
@@ -385,15 +385,18 @@ def put_title_into_background(background, stack_title)
       background
     end
 
-    def actual_font_size
+    def actual_font_size(key)
+      font_size = @config[key.to_s]['font_size']
```

Comment:
```suggestion
      font_size = (@config[key.to_s] ||  {})['font_size']
```

Can we add an extra safety check for this? 👆 I don’t know if that key will also be there so just want to make sure there is an empty hash incase its not 😇

Reply:
Okay, I will add that. I'm no ruby dev, but looking at all other `@config[xxx]` subscripts in the file we never check for that (even in the cases we use the same key `@config[key.to_s]`). Why should we treat this one differently?

Reply:
@ChaosCoder Ah, really?! The kind of scares me a little bit but 🤷‍♂️ I’m not as familiar with the `frameit` codebase so I wasn’t aware that is how the other areas accessed this values. Its probably fine then to keep the code what you had 😇